### PR TITLE
Adds Reset your password done page

### PR DESCRIPTION
**Pull request by Moejoe90**
image
image
image

### CODE REVIEW

File: app/components/walletSelector/walletLight/resetPasswordSuccsess.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+export const WalletLoginHelp: React.SFC<void> = () => (
+  <div>
+    <LayoutRegisterLogin>
+      <Col className="mt-4 pb-5">
+        <WalletResetHeader steps={5} currentStep={5} text={"Lorem ipsum"} />
+      </Col>
+      <Col className="mt-4 mb-5 mx-auto">
+        <h5 className="text-center">
+          {" "}
```
Comment (banciur) [inline app/components/walletSelector/walletLight/resetPasswordSuccsess.tsx]: Why we need space here?

Comment (Moejoe90) [reply app/components/walletSelector/walletLight/resetPasswordSuccsess.tsx]: Should be taken off

File: app/components/walletSelector/walletLight/resetPasswordSuccsess.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import * as React from "react";
+import { Col, Row } from "reactstrap";
+import { LayoutRegisterLogin } from "../../layouts/LayoutRegisterLogin";
+import { StandardButton } from "../../shared/StandardButton";
+import { WalletResetHeader } from "../../walletSelector/walletLight/WalletResetHeader";
+
+export const WalletLoginHelp: React.SFC<void> = () => (
+  <div>
+    <LayoutRegisterLogin>
```
Comment (krzkaczor) [inline app/components/walletSelector/walletLight/resetPasswordSuccsess.tsx]: looks good. But when you integrate everything together you should put layout on top and do not repeat it everywhere.

File: app/components/walletSelector/walletLight/resetPasswordSuccsess.tsx (hunk 1 above)
Comment (krzkaczor) [reply app/components/walletSelector/walletLight/resetPasswordSuccsess.tsx]: this

File: app/components/walletSelector/walletLight/resetPasswordSuccsess.tsx (hunk 2 above)
Comment (Moejoe90) [reply app/components/walletSelector/walletLight/resetPasswordSuccsess.tsx]: that's the plan

**Pull request merged**